scripts/shakeitoff.coffee

```diff
@@ -0,0 +1,104 @@
+# Description:
+# Example scripts for you to examine and try out.
+#
+# Notes:
+# They are commented out by default, because most of them are pretty silly and
+# wouldn't be useful and amusing enough for day to day huboting.
+# Uncomment the ones you want to try and experiment with.
+#
+# These are from the scripting documentation: https://github.com/github/hubot/blob/master/docs/scripting.md
+module.exports = (robot) ->
+robot.hear /shake it off/i, (msg) ->
+msg.send "https://www.youtube.com/watch?v=nfWlot6h_JM"
```

Comment:
It might be nice to add a time tag so it goes straight to "shake it off! sh-sh-shake it off!" in the video